### PR TITLE
feat: add icons for rvm and rspec

### PR DESCRIPTION
**Pull request by r3nya**
Issue #41 

Added icons for rvm stuff and rspec.

### CODE REVIEW

File: src/icons.json [hunk 1]
```diff
@@ -810,22 +810,29 @@
...
-          ".rvmrc"
         ],
-        "base": "source.ruby",
-        "scope": "source.ruby.rvm"
+        "base": "text.plain",
+        "scope": "text.plain.ruby"
       },
       {
-        "name": "Ruby (Rspec)",
+        "name": "Plain Text (Rspec)",
```
Comment (oivva) [inline src/icons.json]: There is no need in `Plain Text (Rspec)` just add the extension to the `Plain Text (Ruby)`.

Comment (oivva) [inline src/icons.json]: In both case we use `Plain Text` and alias to apply the icon. So we need only the one alias for that.

Comment (r3nya) [inline src/icons.json]: Ok, one moment. :)

Comment (r3nya) [inline src/icons.json]: Done!

**Comment by oivva** [comment]
Looks good :wink:

**Comment by r3nya** [comment]
This PR special for @kugaevsky … Dude please wait for our release. 😉

**Pull request merged**